### PR TITLE
PACA-351: Swap language from PreAdjudicated to PartiallyAdjudicated

### PR DESCRIPTION
**Pull request by KyRobbins**
### Fixes [PACA-351](https://jira.cms.gov/browse/PACA-351)

Renaming references to "pre-adjudicated" with "partially-adjudicated" to align with the terminology the PACA team is now using.

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [ ] no PHI/PII is affected by this change

### CODE REVIEW

File: bcda/cclf/testutils/cclfUtils.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func ImportCCLFPackage(acoSize, environment string, fileType models.CCLFFileType
 		"improved-dev":   {"dev", "A9989"},
 		"improved-small": {"small", "A9998"},
 		"improved-large": {"large", "A9999"},
-		"preadj-dev":    {"dev", "TEST001"},
+		"partadj-dev":    {"dev", "TEST001"},
```
Comment (ericbuckley) [inline bcda/cclf/testutils/cclfUtils.go:60]: let's spell out these references as well "partially-adjudicated-dev"

**Review by ericbuckley — APPROVED** [review]
👍

**Review by brianburton — APPROVED** [review]
Looks good.  I don't see any files that were missed.

**Review by katyasoup — APPROVED** [review]

**Review by CosmicSyntax — APPROVED** [review]
🎉

**Pull request merged**